test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/SingleDisposalsTriageControllerSpec.scala

```diff
@@ -4057,7 +4079,7 @@ object SingleDisposalsTriageControllerSpec extends Matchers {
     completeSingleDisposalTriageAnswers: CompleteSingleDisposalTriageAnswers,
     userType: Option[UserType],
     doc: Document
-  )(implicit messages: MessagesApi, lang: Lang): Unit = {
+  )(implicit messagesApi: MessagesApi, messages: Messages, lang: Lang): Unit = {
     completeSingleDisposalTriageAnswers.individualUserType.foreach { individualUserType =>
```

Comment:
you could just leave it as it was and construct the implicit `Messages` in here to avoid having to construct one everywhere else

Reply:
Done.